CHANGES.rst

```diff
@@ -6,9 +6,14 @@ There's a frood who really knows where his towel is.
 2.1b3 (unreleased)
```

Comment:
As already told, we think a 2.1b3 release should be done with what's on master, and this PR a new release, 2.2b1.